src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.Value.cs

```diff
@@ -22,21 +22,19 @@ public sealed partial class PlatformCompatibilityAnalyzer
     {
         private readonly struct PlatformMethodValue : IAbstractAnalysisValue, IEquatable<PlatformMethodValue>
         {
-            private PlatformMethodValue(string invokedPlatformCheckMethodName, string platformPropertyName, Version version, bool negated)
+            internal PlatformMethodValue(string platformPropertyName, Version version, bool negated)
             {
-                InvokedMethodName = invokedPlatformCheckMethodName ?? throw new ArgumentNullException(nameof(invokedPlatformCheckMethodName));
                 PlatformName = platformPropertyName ?? throw new ArgumentNullException(nameof(platformPropertyName));
                 Version = version ?? throw new ArgumentNullException(nameof(version));
                 Negated = negated;
             }
 
-            public string InvokedMethodName { get; }
```

Comment:
This property no really used in any logic except `ToString(), Equals()` overloads